instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -41,8 +41,8 @@
 API
 ---
 """
```

Comment:
Could you add an example usage of the hooks in the docstrings?

Reply:
done.